stdlib/public/core/String.swift

```diff
@@ -968,3 +968,102 @@ internal func _fastWithNormalizedCodeUnitsImpl(
     }
   }
 }
+
+/* Conforms String class to NSString functions, "integerValue", "doubleValue", "floatValue"
+ * Allow for dot notation string to number conversion
+ * Currently to transform a string to a number we must cast using Float(string) which returns an optional
+ * In Obj-c the same conversoin would return 0.0, which may be much more convenient for a more swift development process
+ */
+extension String {
+    
+    // Returns an integer value of the string
+    //  If the value can not be an integer returns 0 instead
+    public var intValue: Int {
+        get {
+            if let value = Int(self) {
+                return value;
+            }
+            else {
+                return 0;
+            }
+        }
+    }
+    
+    // Returns a short value of the string
+    //  If the value can not be a short returns 0 instead
+    public var int8Value: Int8 {
+        get {
+            if let value = Int8(self) {
+                return value;
+            }
+            else {
+                return 0;
+            }
+        }
+    }
+    
+    // Returns a int16 value of the string
+    //  If the value can not be an int16 returns 0 instead
+    public var int16Value: Int16 {
+        get {
+            if let value = Int16(self) {
+                return value;
+            }
+            else {
+                return 0;
+            }
+        }
+    }
+    
+    // Returns a int32 value of the string
+    //  If the value can not be an int32 returns 0 instead
```

Comment:
The indentation needs to be fixed here. Also, you need to use triple-slashes.

I also think the docs could be better written. Maybe use the same ones that NSString has.

Reply:
Appreciate the feedback, i definitely wouldn't considered myself someone who is amazing at writing documentation. I also agree it would be smart to simplify.